src/components/Alert/InlineAlert.tsx

```diff
@@ -19,7 +20,7 @@ interface AlertCardProps {
 const AlertCard: React.FC<AlertCardProps> = ({ children }) => {
   const classes = useStyles({});
 
-  return <Card className={classes.container}>{children}</Card>;
+  return <DashboardCard className={classes.container}>{children}</DashboardCard>;
```

Comment:
question: Do you need this extra class? Couldn't you migrate those styles if you need them?